converters/fromZigbee.js

```diff
@@ -1391,6 +1391,39 @@ const converters = {
             return result;
         },
     },
+    tuya_cover_options: {
+        cluster: 'closuresWindowCovering',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {};
+            if (msg.data.hasOwnProperty('tuyaMovingState')) {
+                const value = msg.data['tuyaMovingState'];
+                const movingLookup = {0: 'up', 1: 'stop', 2: 'down'};
```

Comment:
Please make `UP`, `STOP`, `DOWN` and `ON`/`OFF` below upper case.

Reply:
Done